qutebrowser/config/configdata.yml

```diff
@@ -1622,6 +1622,14 @@ url.yank_ignored_parameters:
     - utm_content
   desc: URL parameters to strip with `:yank url`.
 
+url.yank_remove_password:
```

Comment:
Is this new setting being used anywhere?

Reply:
no, this was supposed to be for something else. I'll remove this again